src/NUnitFramework/tests/Internal/TestMethodSignatureTests.cs

```diff
@@ -84,7 +84,9 @@ public void TestMethodWithWrongNumberOfArgumentsProvidedIsNotRunnable()
         [Test]
         public void TestMethodWithWrongArgumentTypesProvidedGivesError()
         {
-            TestAssert.IsRunnable(fixtureType, "TestMethodWithWrongArgumentTypesProvided", ResultState.Error);
+            // TODO: Should conversion failure somehow differs from the other alignment errors?
```

Comment:
I lean towards yes if it doesn't overcomplicate things. @nunit/framework-team what do you think about this?